app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] @Yosefgeda great work adding associations in the post model, however, you reference the user wrongly, instead you should use the class and author eg ``belongs_to :author, class_name: 'User'``